Explorer++/Explorer++/RegistrySettings.cpp

```diff
@@ -145,6 +145,9 @@ LONG Explorerplusplus::SaveSettings(void)
 		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("LargeToolbarIcons"),m_bLargeToolbarIcons);
 		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("PlayNavigationSound"),m_bPlayNavigationSound);
 
+		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("DisplayMixedFilesAndFolders"),m_bDisplayMixedGlobal);
+		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("SortNamesInNonLogicalOrder"),m_bSortNonLogicalGlobal);

```

Comment:
Move the strings "DisplayMixedFilesAndFolders" and "SortNamesInNonLogicalOrder" into constants. I know the rest of the code doesn't use constants, but enforcing better standards for new code is always good.